server/base/src/main/java/org/apache/accumulo/server/util/UpgradeUtil.java

```diff
@@ -78,10 +78,12 @@ static class Opts extends ConfigOpts {
     boolean prepare = false;
 
     @Parameter(names = "--start",
-        description = "Start an upgrade of an Accumulo instance. This will check that 'accumulo upgrade --prepare'"
+        description = "Start an upgrade of an Accumulo instance. The 'start' step is intended to be run on the "
```

Comment:
Seems fine, but I recommend using the multi-line string literal to make this more readable in the code, now that we are using Java 17 for the main branch.

Reply:
I would have newlines that I do not want in that case.  I could replace them.

Reply:
You can end the line with backslash to avoid introducing unwanted newlines.